plugins/rabbitmq.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: rabbitmq
+spec:
+  homepage: https://github.com/rabbitmq/cluster-operator
+  shortDescription: Manage RabbitMQ clusters
```

```suggestion
  shortDescription: Manage RabbitMQ operator
```
IIUC, this plugin is about interacting with the operator and not with RabbitMQ itself.

---

This plugin has one command `install-cluster-operator` which installs the operator.
Most other commands are used to interact with the RabbitMQ clusters (the custom resources deployed by the operator) directly. 
For example `manage` opens the Management UI for a RabbitMQ cluster, `debug` enables debug logging for a RabbitMQ cluster, `enable-all-feature-flags` enables feature flags for a RabbitMQ cluster.
Other commands, e.g. `create`, `list`, and `delete` interact more with the operator but might arguably also be part of "managing" RabbitMQ clusters? 🤔
I would prefer `Manage RabbitMQ clusters` but would also be happy to rename to `Manage RabbitMQ operator`.

---

Ok, thanks for explaining.
My main concern about `cluster` is that it is a heavily overloaded term. In particular, the k8s folks usually think about the *kubernetes* cluster as a whole and less about a rabbitmq cluster. Given the circumstances, it looks this ambiguity is inevitable.